components/open-map-button.tsx

```diff
@@ -0,0 +1,25 @@
+import { SecondaryButton } from "./ui/button";
+
+import { MapIcon } from "@heroicons/react/outline";
+
+export function OpenMapButton({ address }: { address: string }) {
+  function btnClick(e: React.FormEvent<HTMLButtonElement>) {
+    e.currentTarget.blur();
+    window.open(`https://www.google.com/maps/place/${address}`);
```

Comment:
Please use `https://www.google.com/maps/search` instead of `https://www.google.com/maps/place`

Reply:
@ekamuktia resolved, thanks